### ISSUE TITLE
The pugling doesn't work in my website

### ISSUE THREAD

**Issue by mvarsuarez** [dsifford/academic-bloggers-toolkit#259]
**ABT Version:** 4.8.0

**PHP Version:** 5.6

Relia Pro

 I should confess that I have a lot of plugins installed. I try to remove as many of them as possible, but I cannot find which is in conflict with ABT. Actually, I want to have this plugin active in my website because it's an academic website and the plugin is perfect for it (thank you for created it). 
**Plugin List:**
- Black Studio TinyMCE
- TinyMCE Advanced
- Custom Login
- Easy Google Fonts
- Flexible posts widget
- Forget About Shortcode Buttons
- GS Team
- iThemeland Logo Showcase for wp
- Pagebuilder for SiteOrigin
- Rename wp-login.php
- Spacer
- Ultimate member
- Ultimate Member - Navigation Menu
- Social Media Links for Ultimate Member
- Team members
- VM Menu Reorder plugin
- WP News slider
- Widgets for SiteOrigin
- Wassup
- Wordpress Image Hover Lite 
- WP Mega Menu
- WP Team Showcase and Slider
- WPBizPlugins Custom Admin Help Boxes

**Browser:** Google Chrome (Version: xxx)

**Expected behavior:**

**Actual behavior:** When I go the "posts" section I see the reference's widget, but I cannot see the button of the plugin.  

**Console messages:**
```js
// Paste messages here

```

Thank you in advance!

**Comment by dsifford** [comment]
@mvarsuarez It's almost certainly a plugin conflict.

My guess is that it's the SiteOrigins plugin that is causing the problem (see [here](https://github.com/dsifford/academic-bloggers-toolkit/issues/220)). Can you try disabling that plugin to see if that's indeed the case?

Another possible conflict would be the Black Studio TinyMCE plugin. Try disabling that one too.

**Comment by dsifford** [comment]
@mvarsuarez I just confirmed that PageBuilder by SiteOrigin is the issue. This plugin requires the default editor. It cannot be used in conjunction with plugins that modify this behavior.

**Comment by intersub** [comment]
Found that it is possible to use PageBuilder by SiteOrigin with Academic Blogger's Toolkit.  Hack is simply to deactivate some widgets in SiteOrigin Bundle when entering/editing citations with A.B's Toolkit.  Citations persist after widget(s) re-activation, despite not being able to "see the button of the plugin."

1. go to >Plugins>SiteOrigin Widgets 
    (/plugins.php?page=so-widgets-plugins)

2. click > [deactivate] on Editor widget
    (A widget which allows editing of content using the TinyMCE editor.

3. reload post editor.  Academic Blogger's Toolkit works again for me.

This may need to be repeated every time SiteOrigin reactivates the widget, for example when I use PageBuilder again.  Other plugins by SiteOrigin I also deactivated, without diagnosing them individually, the ones otherwise I find do not interfere with A.B's toolkit are:

- Post Carousel
- Image
- Icon
- Headline
- Button

Hope this helps someone.
